deku/deku.d.ts

```diff
@@ -0,0 +1,137 @@
+// Type definitions for deku v2.0.0-rc16
```

Comment:
For a types-2.0 definition, prefer `index.d.ts`.
Also, patch version is ignored -- you just need `2.0`.